llvm-runtimes/libcxx/libcxx-19.1.7.ebuild

```diff
@@ -67,12 +67,15 @@ test_compiler() {
 src_configure() {
 	llvm_prepend_path "${LLVM_MAJOR}"
 
+	local install_prefix=${EPREFIX}
+	is_crosspkg && install_prefix+=/usr/${CTARGET}
```

Comment:
Is this actually correct? Sounds like you'd be installing `/usr/${CTARGET}/usr/...`.

Reply:
Sorry, you were correct. I'm looking at my aarch64 sysroot and indeed it follows this style. I suppose I gotten confused by `/usr/${CHOST}` used by toolchain.